### DISCUSSION TITLE
the question in encoding RGBA frame video

### DISCUSSION THREAD

**Discussion by lianganwei1994** [PyAV-Org/PyAV#848] [1. Help]
Hi I'm using pyav to encode a mov video with rgba frames:
self.container = av.open('temp.mov', mode='w',format = 'mov')
self.stream = self.container.add_stream('prores_ks', rate=self.fps)  
self.stream.framerate = self.fps
self.stream.pix_fmt = "yuva444p10le" 
self.stream.width = 1920
self.stream.height = 1080

but I found that the encoding speed is to low, only 200ms per frame and the generate file is too large
Is there any method to improve enconding speed or any other codec can be used?

self.stream.bit_rate = 200000
the other question is even i set the bit_rate it seem invalid
thanks

**Comment by yiya1989** [comment]
i have the same question，how you solve it?
